### PR TITLE
DX-458

### PR DESCRIPTION
**Pull request by mlsnyder**
hid the fetch results from the model homepage

### CODE REVIEW

File: dynamic/build-models.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const loadFile = function(file) {
 const writeDirectoryIndex = function(dir, defs, product, apiName) {
     let table = `---
 layout: default
-title: "${apiName}"
```
Comment (JeremyCraigMartinez) [inline dynamic/build-models.js]: why did you make this change?

Comment (mlsnyder) [inline dynamic/build-models.js]: That was unintentional. I'll change it back.

**Comment by JeremyCraigMartinez** [comment]
Did you test this locally?

File: dynamic/build-models.js [hunk 2]
```diff
@@ -49,12 +49,15 @@ endpoint_links: []
...
         }
-
-        table = `${table}
-        <tr>
-            <td><a href="${encodeURIComponent(def)}">${def}</a></td>
-            <td>{{"${(defs[def].description || '').replace(/"/g, "'")}" | markdownify}}</td>
-        </tr>`;
+        let model = def;
+
+        if (!model.includes('FetchResults')) {
```
Comment (JeremyCraigMartinez) [inline dynamic/build-models.js]: `FetchResults` should be `FetchResult`

Comment (mlsnyder) [inline dynamic/build-models.js]: Yes, I did test it. Obviously not well enough, I will make this change and test it more throughly.

**Pull request merged**